lib/cext/nokogiri_patches.rb

```diff
@@ -0,0 +1,342 @@
+# Patches nokogiri 1.8.1
```

Comment:
These patches should work for both 1.8.x and 2.0.x. If you think we should split them, we can.